ccmlib/node.py

```diff
@@ -1798,7 +1798,7 @@ def resume(self):
             p.resume()
         except ImportError:
             if common.is_win():
-                print_("WARN: psutil not installed. Resume functionality will not work properly on Windows.")
+                common.warning("WARN: psutil not installed. Resume functionality will not work properly on Windows.")
```

Comment:
Now that we are logging, we no longer want the log level prepended in the message, as that will happen automatically now. Meaning these will now be redundant